### PR TITLE
Business_Goal_US01 : Add professional group navigations

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/group/platform/professional/pages.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+-->
+<page-set xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.gatein.org/xml/ns/gatein_objects_1_3 http://www.gatein.org/xml/ns/gatein_objects_1_3"
+	xmlns="http://www.gatein.org/xml/ns/gatein_objects_1_3">
+	<page>
+		<name>appCenterAdminSetup</name>
+		<title>App center admin setup</title>
+		<access-permissions>*:/platform/professional</access-permissions>
+		<edit-permission>*:/platform/professional</edit-permission>
```
Comment (plamarque) [inline app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/group/platform/professional/pages.xml]: should be *:/platform/administrators IMO

File: app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/group/platform/professional/pages.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+				<preferences>
+					<preference>
+						<name>pageSize</name>
+						<value>10</value>
+						<read-only>false</read-only>
+					</preference>
+				</preferences>
+			</portlet>
+			<title>App Center Admin Setup</title>
+			<access-permissions>*:/platform/professional</access-permissions>
```
Comment (plamarque) [inline app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/group/platform/professional/pages.xml]: shouldn't this be multi-valued in order to also give access to *:/platform/administrators ?

File: app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/group/platform/professional/navigation.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+-->
+<node-navigation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.gatein.org/xml/ns/gatein_objects_1_3 http://www.gatein.org/xml/ns/gatein_objects_1_3"
+	xmlns="http://www.gatein.org/xml/ns/gatein_objects_1_3">
+	<priority>1</priority>
+	<page-nodes>
+		<node>
+			<name>appCenterAdminSetup</name>
+      <label>#{administration.appCenterAdminSetup}</label>
+			<page-reference>group::/platform/professional::appCenterAdminSetup</page-reference>
```
Comment (plamarque) [inline app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/group/platform/professional/navigation.xml]: I'm affraid putting this page inside /platform/professional group nav will prevent other admins (*:/platform/administrators) to access it.

**Review by plamarque — COMMENTED** [review]
I have concerns that choices made in this PR will rpevent normal admins to access the app center admin

File: app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/group/platform/professional/pages.xml (hunk 1 above)
Comment (AzmiTouil) [reply app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/group/platform/professional/pages.xml]: Done

File: app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/group/platform/professional/pages.xml (hunk 2 above)
Comment (AzmiTouil) [reply app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/group/platform/professional/pages.xml]: updated

File: app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/group/platform/professional/navigation.xml (hunk 3 above)
Comment (AzmiTouil) [reply app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/group/platform/professional/navigation.xml]: updated

**Pull request closed without merging**